docs/components/motor/gpio/encoded-motor.md

```diff
@@ -13,6 +13,10 @@ aliases:
 Use an [encoder](/components/encoder/) with a motor to create a closed feedback loop for better control of your machine.
 Instead of sending speed or position commands without a way to verify the motor's behavior, the encoder lets the computer know how the motor is actually rotating in the real world, so adjustments can be made to achieve the desired motor movement.
 
+For example, a [Viam Rover](https://docs.viam.com/get-started/try-viam/rover-resources/) with encoded motors following both an [angular](/components/base/#spin) and [linear](/components/base/#movestraight) velocity command:
+
+{{<video webm_src="/components/encoded-motor/base_moving.webm" mp4_src="/components/encoded-motor/base-moving.mp4" alt="A Viam rover turning in a half circle" poster="components/motor/encoded-motor-preview.jpg">}}
```

Comment:
I think this is good but might fit better in the `Test the motor` section alongside the video that shows how the control view changes? What do you think? https://drive.google.com/file/d/1M6GPnsFo2xZYADi08e1oMEtU3_WSaLgJ/view?resourcekey

Reply:
on it

Reply:
I don't think I see the other video alongside it, do you think that's not worth adding?

Reply:
@npentrel Sorry, I somehow thought you were talking about the screenshot in the control snippet, didn't comprehend it was a separate video. Will add that now. Thanks for catching.